packages/firebase_crashlytics/android/build.gradle

```diff
@@ -37,7 +34,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.crashlytics.sdk.android:crashlytics:2.9.9'
+    implementation 'com.google.firebase:firebase-crashlytics:17.0.0-beta03'
```

Comment:
Looks like they have moved to **beta4** as per the official doco [here](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android)